### DISCUSSION TITLE
Don't use fabric api?

### DISCUSSION THREAD

**Discussion by legoraft** [FabricMC/fabric-api#1642] [Mod Dev Support] [answered]
I want to write a mod that doesn't use the fabric API, but I can't find anything on the internet and I immediately ran into an issue with the modInitializer, because it's part of the fabric API. Does anyone know how to write a mod without the fabric API?

**Comment by modmuss50** [comment, marked as answer]
You can remove it from your build.gradle. 

Fabric API can easily be removed, fabric-loader you will want to keep as that loads your mod and provides the basic modInitializer's.

**Reply by legoraft** [reply]
Thanks a lot! I got an error, but it was because I forgot to remove a dependency
